contracts/script/1_FinalizerAvsDeployer.s.sol

```diff
@@ -19,6 +19,8 @@ import "@eigenlayer-middleware/src/StakeRegistry.sol";
 import {FinalizerServiceManager, IServiceManager} from "../src/FinalizerServiceManager.sol";
 import {FinalizerTaskManager} from "../src/FinalizerTaskManager.sol";
 import {IFinalizerTaskManager} from "../src/IFinalizerTaskManager.sol";
+import {Rolldown} from "../src/Rolldown.sol";
+import {ERC20Mock} from "../src/ERC20Mock.sol";
```

Comment:
what is the purpose of having the erc20mock token here ? it is not used anywhere in the script, just to print out the address at the end

on local anvil chain 31337 we use the mock to setup the eigenlayer with some staking token - in 0_AnvilSetup.s.sol

but on holesky 17000 we only use this script to deploy, no mock token is present there

Reply:
Fixed - dumping strategy_output.json (contains erc20Mock address) into github ci summary